pcsx2/Counters.cpp

```diff
@@ -214,7 +213,7 @@ static void vSyncInfoCalc(vSyncTimingInfo* info, Fixed100 framesPerSecond, u32 s
 	u64 hBlank = Scanline / 2;
 	u64 hRender = Scanline - hBlank;
 
-	if (gsRegionMode == Region_NTSC_PROGRESSIVE)
+	if (gsVideoMode >= GS_VideoMode::VESA)
```

Comment:
> However I rather believe this functions screams for a switch were every mode is explicitly mentioned 

I thought you hated switch-case ? :P 

Either way, a really nice suggestion. I'll change it ASAP.